pkg/cloudprovider/gce/gce.go

```diff
@@ -136,14 +144,15 @@ func GetGCEAttributes(mac string, instance *cloudprovider.CloudInstance) map[res
 	attributes := make(map[resourceapi.QualifiedName]resourceapi.DeviceAttribute)
 	attributes[AttrGCEMachineType] = resourceapi.DeviceAttribute{StringValue: &instance.Type}
 
-	topologyParts := strings.SplitN(strings.TrimPrefix(instance.Topology, "/"), "/", 3)
 	// topology may not be always available
-	if len(topologyParts) == 3 {
-		attributes[AttrGCEBlock] = resourceapi.DeviceAttribute{StringValue: &topologyParts[0]}
-		attributes[AttrGCESubBlock] = resourceapi.DeviceAttribute{StringValue: &topologyParts[1]}
-		attributes[AttrGCEHost] = resourceapi.DeviceAttribute{StringValue: &topologyParts[2]}
-	} else {
-		klog.Warningf("Error parsing host topology %q; it may be unsupported for the VM", instance.Topology)
```

Comment:
we need to make this check better and only log when is indeed a problem of parsing, if it does not exist at all we should not log anything